formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -30,28 +29,27 @@ let tmp = {
 /**
  * Comment on stackWithAttributes.
  */
```

Comment:
I would recommend taking out the special case and making the code simpler to evolve/iterate on.
Curious, with this current special case, how does one distinguish a comment on the first method, vs. on the entire object returned?